src/activeList.js

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-unused-vars */
+/* eslint-disable func-names */
+/* eslint-disable no-restricted-globals */
+/* eslint-disable import/no-mutable-exports */
+
+import _ from 'lodash';
+
+/* eslint-disable no-const-assign */
+const form = document.querySelector('#form');
+const inputF = document.querySelector('.inputF');
+const parentNode = document.querySelector('.parent-node');
+const enterErr = document.querySelector('.enter-err');
+const edit = document.querySelector('.edit');
+const enterBtn = document.querySelector('.btn');
+const removeAll = document.querySelector('.removeAll');
+
+let itemsArr = [];
+
+const removeItem = function (item) {
+  const removeIndex = itemsArr.indexOf(item);
+  itemsArr.splice(removeIndex, 1);
+};
+
+const updateItem = function (currentItemIndex, value) {
+  const newItem = itemsArr[currentItemIndex];
+  newItem.description = value;
+  itemsArr.splice(currentItemIndex, 1, newItem);
+  // eslint-disable-next-line no-use-before-define
+  saveLocalStorage(itemsArr);
+};
+
+// eslint-disable-next-line func-names
+const handleItem = function (itemData) {
+  const items = document.querySelectorAll('.show-list');
+  items.forEach((item) => {
+    // eslint-disable-next-line eqeqeq
+    if (item.querySelector('.tast').getAttribute('data-time') == itemData.index) {
```

Comment:
I think it's better to use `=== (Triple equals operator)` here

```suggestion
    if (item.querySelector('.tast').getAttribute('data-time') === itemData.index) {
```